README.md

```diff
@@ -1 +1,27 @@
-# Relationship between social distancing measure and impact of covid-19
+# The relationship between social distancing measure and impact of covid-19: a collaborative research project
+
+
+
+## Hi! :wave:
+
+Welcome! This repository is for an ongoing collaborative project by members of the Health Data Lab team from the [Centre for Health Data Science](https://www.abdn.ac.uk/achds/), part of the University of Aberdeen.
+
+We are conducting research on the relationship between the [Grampian Social Distancing Score](https://jessbutler.github.io/social_distancing/#social-distancing-score), which was developed by [@JessButler](https://github.com/JessButler) to identify areas that might be at greater risk of transmitting COVID-19 due to likely constraints in the ability to adhere to social distancing guidelines.
```

Comment:
I would say (it's wordier but clearer I think?): We are conducting research on the relationship between the [Grampian Social Distancing Score](https://jessbutler.github.io/social_distancing/#social-distancing-score) and the impact of COVID-19. The Grampian Social Distancing Score was developed by [@JessButler](https://github.com/JessButler) to identify areas that might be at greater risk of transmitting COVID-19 due to likely constraints in the ability to adhere to social distancing guidelines.

Reply:
Thanks Dimitra. That sounds reasonable to me - If I update that on my local repo (in R) and commit/push, will it automatically update this pull request?

Reply:
Yes it will!

Reply:
Done!

Reply:
Sorry, there's a bit missing still. "We are conducting research on the relationship between GSDS" ... and what? "and the impact of COVID-19"

Reply:
😄 sorry, just done that too